sphinxcontrib/spelling/builder.py

```diff
@@ -204,9 +204,9 @@ def _find_misspellings(self, docname, doctree):
 
                 # Check the text of the node.
                 misspellings = self.checker.check(node.astext())
-                for word, suggestions, context_line in misspellings:
+                for word, suggestions, context_line, line_offs in misspellings:
```

Comment:
Since you'll need to update this PR with a change in `history.rst` anyway, could I get you to rename `line_offs` to `line_offset`?